cs/test/RunTestsHelper.cs

```diff
@@ -0,0 +1,129 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VW;
+
+namespace cs_test
```

Comment:
did you move this? wasn't this always there?

Reply:
Not needed.  I added a copy to isolate and test some functionality.